test/unit/lib/plugins/aws/package/compile/events/cognito-user-pool.test.js

```diff
@@ -802,4 +963,426 @@ describe('AwsCompileCognitoUserPoolEvents', () => {
       ).to.equal('AWS::Lambda::Permission');
     });
   });
+
+  describe('Custom Sender Sources', () => {
+    describe('Schema Issues', () => {
+      it('should throw if more than 1 KMS Key is configured per new Cognito User Pool', () => {
+        return expect(
+          runServerless({
+            fixture: 'function',
+            configExt: {
+              functions: {
+                first: {
+                  handler: 'index.js',
```

Comment:
I don't think you have to override handler here

Reply:
To eliminate this, I overrode the `basic` function defined in the `function` fixture, instead of creating a new function.

The function is declared in the `function` fixture here: https://github.com/serverless/serverless/blob/main/test/fixtures/programmatic/function/serverless.yml#L10